boot/boot_serial/src/boot_serial.c

```diff
@@ -397,8 +415,12 @@ bs_upload(char *buf, int len)
                     goto out;
                 }
             }
-        }
 #endif
+#ifdef MCUBOOT_IMAGE_ACCESS_HOOKS
+            rc = boot_serial_uploaded_hook(img_num, fap, img_size);
```

Comment:
The `rc` is not validated anywhere, the static analysis tool will be poking us for that.

Reply:
now is treated as operation error